src/main/java/com/iexec/core/task/update/TaskUpdateRequestManager.java

```diff
@@ -63,14 +79,23 @@
             if (chainTaskId.isEmpty()){
                 return false;
             }
-            if (queue.contains(chainTaskId)){
+            if (queue.containsTask(chainTaskId)){
                 log.warn("Request already published [chainTaskId:{}]", chainTaskId);
                 return false;
             }
-            boolean isOffered = queue.offer(chainTaskId);
+            final Optional<Task> task = taskService.getTaskByChainTaskId(chainTaskId);
+            if (task.isEmpty()) {
+                log.warn("No such task. [chainTaskId: {}]", chainTaskId);
+                return false;
+            }
+
+            taskUpdateExecutor.execute(new TaskUpdate(task.get(), locks, consumer));
             log.info("Published task update request [chainTaskId:{}, queueSize:{}]", chainTaskId, queue.size());
```

Comment:
If we want to keep that log, we could improve them by adding taskUpdate status & deadline

Reply:
Sure, that could be handy. What about making that log `debug` instead of `info`?

Reply:
Yes